client/components/authentication/UserInfo.js

```diff
@@ -48,14 +48,14 @@ class UserInfo extends Component {
       username, password, firstName, lastName, userEmail,
     } = this.state;
     const {
-      type, login, createUser, user,
+      type, login, createUser,
```

Comment:
Looks like we can add user back to these deconstructed variables since you call this.props.user.id on line 58